devito/types/dimension.py

```diff
@@ -722,6 +722,7 @@ class ConditionalDimension(DerivedDimension):
     def __init_finalize__(self, name, parent, factor=None, condition=None,
                           indirect=False):
         super().__init_finalize__(name, parent)
+
```

Comment:
is this blank line homogeneous w.r.t. the other constructors in this module?

Reply:
Sorry, a leftover. Dropped.